src/Parser/Evaluated/Rule/Natural/FeaturedSnipped.php

```diff
@@ -97,7 +97,13 @@ public function version2(GoogleDom $googleDOM, \DomElement $node, IndexedResultS
         $aTag = $googleDOM->getXpath()->query("descendant::a", $node);
         $description = $googleDOM->getXpath()->query("descendant::div[@class='LGOjhe']", $node);//description
 
-        if (!empty($aTag) && !empty($aTag->item(0))) {
+        if (!empty($aTag) && !empty($aTag->item(1))) {
```

Comment:
did you validate this with an "older" position zero fix? where the serp is on the first anchor tag position? ($aTag->item(0)).
maybe you could check if the href attribute is a valid url and use this test to identify the item position.